core/data_migration_scripts/EE_DMS_Core_4_6_0.dms.php

```diff
@@ -543,18 +543,18 @@ public function add_default_admin_only_payments()
             $default_admin_only_payment_methods = apply_filters(
                 'FHEE__EEH_Activation__add_default_admin_only_payments__default_admin_only_payment_methods',
                 array(
-                    __("Bank", 'event_espresso')        => __("Bank Draft", 'event_espresso'),
-                    __("Cash", 'event_espresso')        => __("Cash Delivered Physically", 'event_espresso'),
-                    __("Check", 'event_espresso')       => __("Paper Check", 'event_espresso'),
-                    __("Credit Card", 'event_espresso') => __("Offline Credit Card Payment", 'event_espresso'),
-                    __("Debit Card", 'event_espresso')  => __("Offline Debit Payment", 'event_espresso'),
-                    __("Invoice", 'event_espresso')     => __(
+                    (string) __("Bank", 'event_espresso')        => __("Bank Draft", 'event_espresso'),
+                    (string) __("Cash", 'event_espresso')        => __("Cash Delivered Physically", 'event_espresso'),
+                    (string) __("Check", 'event_espresso')       => __("Paper Check", 'event_espresso'),
+                    (string) __("Credit Card", 'event_espresso') => __("Offline Credit Card Payment", 'event_espresso'),
+                    (string) __("Debit Card", 'event_espresso')  => __("Offline Debit Payment", 'event_espresso'),
+                    (string) __("Invoice", 'event_espresso')     => __(
                         "Invoice received with monies included",
                         'event_espresso'
                     ),
-                    __("Money Order", 'event_espresso') => '',
-                    __("Paypal", 'event_espresso')      => __("Paypal eCheck, Invoice, etc", 'event_espresso'),
-                    __('Other', 'event_espresso')       => __('Other method of payment', 'event_espresso'),
+                    (string) __("Money Order", 'event_espresso') => '',
+                    (string) __("Paypal", 'event_espresso')      => __("Paypal eCheck, Invoice, etc", 'event_espresso'),
+                    (string) __('Other', 'event_espresso')       => __('Other method of payment', 'event_espresso'),
```

Comment:
yikes, using translations as array keys seems super fragile.  I'm also wondering how well this works across php versions.  I understand there's some limitations on what we can do here for back compat reasons...

Reply:
I suppose one problem with using translated strings as array keys is in case the translations are coincidentally the same. Yeah, I should have just used sub-arrays with 2 elements instead. But ya, I don't think we can change this without breaking backwards compatibility